java/piranha/src/test/java/com/uber/piranha/CorePiranhaTest.java

```diff
@@ -1028,4 +1036,198 @@ public void testRemoveSpecificAPIpatterns() throws IOException {
             "}")
         .doTest();
   }
+
+  public void transformAndCreateNewPropertyFile(
+      String srcProp, String trgtProp, String stale_flag, boolean isTreated) throws IOException {
+
+    String[] temp = stale_flag.split("_");
+    String methodPropertiesToAdd =
+        "{\n"
+            + "      \"methodName\": \"$flagName$.getValue\",\n"
+            + "      \"flagType\": \"treated\",\n"
```

Comment:
Always treat this series of invocation as treated. So that it will evaluate to `true` when `isTreated = true` else `false`